### DISCUSSION TITLE
Open Collective and Inflation

### DISCUSSION THREAD

**Discussion by sandsalamand** [stride3d/stride#1973] [Polls]
The Open Collective page shows signs of stagnation. Given that the USD inflation is now at 5-10%, I think it's imperative that the Collective starts splashing out some major cash to get the ball rolling. This "slow and steady" approach of saving up money doesn't seem to be working, and, when inflation is taken into account, the Collective might even be losing money each year.

A lot of potential new features would be aided by the existence of a plugin system, and Linux support is crucial for acquiring a larger base of skilled developers. Therefore, I would suggest that the Collective offers a large sum of money, somewhere between $5,000-$10,000, for the purpose of building the bones of a plugin-oriented Avalonia editor. The goal would be to have the level of flexibility of the Godot editor.

I'm aware that this amount of money is chump change for skilled developers, so the community will still have to do some serious work in building upon the system. However, if it's built with extensibility in mind, I think many features could be implemented piece-by-piece. For example, even something as fundamental as the asset browser could be designed as a plugin.

**Comment by Eideren** [comment]
We do want to spend the cash, issue is finding someone to work on those features. Aggror already spent a considerable amount of time trying to find people to work on those features, and it was mostly a waste of time on his end.
As we specified on the readme of this repo, if you want to work and be paid for any feature, we're open to it, but we just don't have time to chase talent. We could expand the idea to any kind of issue instead of those logged in the open collective, but we would still need users to be proactive here.

**Reply by sandsalamand** [reply]
That's good to hear, but the Open Collective page still doesn't have this listed as an initiative. Why are we only searching through private channels, instead of advertising it with some big dollar signs on the page?

**Reply by Eideren** [reply]
Feel free to open a PR for the website or this repo for anything you think would help. I'll check back with the team for the changes to the open collective
